<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yeastEditor</class>
 <widget class="QDialog" name="yeastEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yeast Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget_editor">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_main">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_name</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="SmartLineEdit" name="lineEdit_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_type">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_yeastType</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="BtComboBox" name="comboBox_yeastType">
         <property name="toolTip">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_amountIsWeight">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Check it if the amount given is in kg instead of L.</string>
         </property>
         <property name="text">
          <string>Amount is weight?</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="checkBox_amountIsWeight">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Checked if the given amount is weight instead of volume</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_form">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Form</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_yeastForm</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="BtComboBox" name="comboBox_yeastForm">
         <property name="toolTip">
          <string>Form</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="SmartLabel" name="label_inventory">
         <property name="text">
          <string>Total amount in Inventory</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_inventory</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="SmartLineEdit" name="lineEdit_inventory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Total amount in inventory</string>
         </property>
        </widget>
       </item>

       <item row="2" column="4">
        <widget class="QLabel" name="label_amountType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Whether the amount is weight or volume</string>
         </property>
         <property name="text">
          <string>Amount Type</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_amountType</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="BtComboBox" name="comboBox_amountType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Whether the amount is weight or volume</string>
         </property>
        </widget>
       </item>

       <item row="3" column="0">
        <widget class="QLabel" name="label_laboratory">
         <property name="text">
          <string>Lab</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_laboratory</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="SmartLineEdit" name="lineEdit_laboratory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Lab</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_productId">
         <property name="text">
          <string>Product ID</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_productId</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="SmartLineEdit" name="lineEdit_productId">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Product ID</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_extra">
      <attribute name="title">
       <string>Extras</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_attenuation">
         <property name="text">
          <string>Attenuation (%)</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_attenuation</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="SmartLineEdit" name="lineEdit_attenuation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Apparent attenuation as percentage of OG points</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_flocculation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Flocculation</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="BtComboBox" name="comboBox_yeastFlocculation">
         <property name="toolTip">
          <string>Flocculation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_attenuationMin">
         <property name="text">
          <string>Min Attenuation (%)</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_attenuationMin</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="SmartLineEdit" name="lineEdit_attenuationMin">
         <property name="toolTip">
          <string>Minimum apparent attenuation as percentage of OG points</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_attenuationMax">
         <property name="text">
          <string>Max Attenuation (%)</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_attenuationMax</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="SmartLineEdit" name="lineEdit_attenuationMax">
         <property name="toolTip">
          <string>Maximum apparent attenuation as percentage of OG points</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="SmartLabel" name="label_minTemperature">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Min Temp</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_minTemperature</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="SmartLineEdit" name="lineEdit_minTemperature">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Min temp</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_maxReuse">
         <property name="text">
          <string>Max Recultures</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_maxReuse</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="SmartLineEdit" name="lineEdit_maxReuse">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Max recultures</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="SmartLabel" name="label_maxTemperature">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Max Temp</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_maxTemperature</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="SmartLineEdit" name="lineEdit_maxTemperature">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Max temp</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_timesCultured">
         <property name="text">
          <string>Times Recultured</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_timesCultured</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="SmartLineEdit" name="lineEdit_timesCultured">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Times this yeast has been recultured</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_addToSecondary">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Add to Secondary</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="BtBoolComboBox" name="boolCombo_addToSecondary">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Whether to add this yeast to secondary instead of primary</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_alcoholTolerance">
         <property name="text">
          <string>Alcohol Tolerance</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_alcoholTolerance</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="SmartLineEdit" name="lineEdit_alcoholTolerance">
         <property name="toolTip">
          <string>Recommended limit of ABV by the culture producer before attenuation stops</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_phenolicOffFlavorPositive">
         <property name="text">
          <string>Phenolic Off-Flavor Positive</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="BtBoolComboBox" name="boolCombo_phenolicOffFlavorPositive">
         <property name="toolTip">
          <string>Whether culture is capable of producing phenols, which is a common distinctive property of saison, and brett yeasts</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_glucoamylasePositive">
         <property name="text">
          <string>Glucoamylase Positive</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="BtBoolComboBox" name="boolCombo_glucoamylasePositive">
         <property name="toolTip">
          <string>A glucoamylase positive culture is capable of producing glucoamylase, the enzyme produced through expression of the diastatic gene, which allows yeast to attenuate dextrins and starches leading to a very low FG. This is positive in some saison/brett yeasts as well as the new gulo hybrid by Omega yeast labs.</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_killerProducingK1Toxin">
         <property name="text">
          <string>K1 Killer</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="BtBoolComboBox" name="boolCombo_killerProducingK1Toxin">
         <property name="toolTip">
          <string>Produces K1 killer toxin</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="label_killerProducingK2Toxin">
         <property name="text">
          <string>K2 Killer</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="BtBoolComboBox" name="boolCombo_killerProducingK2Toxin">
         <property name="toolTip">
          <string>Produces K2 killer toxin</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_killerProducingK28Toxin">
         <property name="text">
          <string>K28 Killer</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="BtBoolComboBox" name="boolCombo_killerProducingK28Toxin">
         <property name="toolTip">
          <string>Produces K28 killer toxin</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="label_killerProducingKlusToxin">
         <property name="text">
          <string>Klus Killer</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="BtBoolComboBox" name="boolCombo_killerProducingKlusToxin">
         <property name="toolTip">
          <string>Produces Klus killer toxin</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_killerNeutral">
         <property name="text">
          <string>Killer Neutral</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="BtBoolComboBox" name="boolCombo_killerNeutral">
         <property name="toolTip">
          <string>Neutral strains do not produce toxins, nor are they killed by them</string>
         </property>
        </widget>
       </item>

       <item row="9" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_bestfor">
      <attribute name="title">
       <string>Best For</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit_bestFor"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_notes">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEdit_notes"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_new">
       <property name="toolTip">
        <string>New yeast</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/smallPlus.svg</normaloff>:/images/smallPlus.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="toolTip">
        <string>Save and close</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/filesave.svg</normaloff>:/images/filesave.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="toolTip">
        <string>Discard and close</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/exit.svg</normaloff>:/images/exit.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/BtComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>BtBoolComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/BtBoolComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>SmartLabel</class>
   <extends>QLabel</extends>
   <header>widgets/SmartLabel.h</header>
   <slots>
    <signal>changedSystemOfMeasurementOrScale(PreviousScaleInfo)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>SmartLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/SmartLineEdit.h</header>
   <slots>
    <slot>lineChanged(PreviousScaleInfo)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget_editor</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>comboBox_yeastType</tabstop>
  <tabstop>comboBox_yeastForm</tabstop>
  <tabstop>lineEdit_laboratory</tabstop>
  <tabstop>lineEdit_productId</tabstop>
  <tabstop>lineEdit_inventory</tabstop>
  <tabstop>lineEdit_attenuation</tabstop>
  <tabstop>lineEdit_minTemperature</tabstop>
  <tabstop>lineEdit_maxTemperature</tabstop>
  <tabstop>comboBox_yeastFlocculation</tabstop>
  <tabstop>lineEdit_maxReuse</tabstop>
  <tabstop>lineEdit_timesCultured</tabstop>
  <tabstop>boolCombo_addToSecondary</tabstop>
  <tabstop>pushButton_new</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>textEdit_bestFor</tabstop>
  <tabstop>textEdit_notes</tabstop>
 </tabstops>
 <resources>
  <include location="../brewken.qrc"/>
 </resources>
</ui>
